packages/react-native/ReactCommon/react/renderer/components/view/BaseViewProps.cpp

```diff
@@ -166,6 +166,14 @@ BaseViewProps::BaseViewProps(
                                                        "experimental_filter",
                                                        sourceProps.filter,
                                                        {})),
+      backgroundImage(
+          CoreFeatures::enablePropIteratorSetter ? sourceProps.backgroundImage
```

Comment:
Please also add to the area around `RAW_SET_PROP_SWITCH_CASE_BASIC`

`enablePropIteratorSetter` is still enabled in a few places internally, and if enabled, it goes through that path, in `setProp`, instead of the `convertRawProp` here.